### PR TITLE
[plugins] Disables the AndroidGradlePluginVersion issue ID in all android packages

### PR DESCRIPTION
**Pull request by FXschwartz**
Disables the `AndroidGradlePluginVersion` issue ID in each Android package to prevent a gradle version warning from showing up in Github UI.

*List which issues are fixed by this PR. You must list at least one issue.*
Fixes - https://github.com/flutter/flutter/issues/95048

*If you had to change anything in the [flutter/tests] repo, include a link to the migration guide as per the [breaking change policy].* N/A

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [relevant style guides] and ran [the auto-formatter]. (Unlike the flutter/flutter repo, the flutter/plugins repo does use `dart format`.)
- [x] I signed the [CLA].
- [x] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. `[shared_preferences]`
- [x] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy], or this PR is [exempt from version changes].
- [x] I updated `CHANGELOG.md` to add a description of the change, [following repository CHANGELOG style].
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/plugins/blob/main/CONTRIBUTING.md
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[relevant style guides]: https://github.com/flutter/plugins/blob/main/CONTRIBUTING.md#style
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[exempt from version changes]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#version-and-changelog-updates
[following repository CHANGELOG style]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style
[the auto-formatter]: https://github.com/flutter/plugins/blob/main/script/tool/README.md#format-code
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests

### CODE REVIEW

**Comment by Hixie** [comment]
test-exempt: configuration change

**Comment by stuartmorgan** [comment]
Overriding version and changelog checks, since these are dev-only changes and thus exempt per repo policy.

File: packages/camera/camera_android/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## NEXT
+
+* Disables the AndroidGradlePluginVersion issue ID in all android packages.
```
Comment (stuartmorgan) [inline packages/camera/camera_android/CHANGELOG.md:3]: These changelog changes can all be reverted.

**Review by stuartmorgan — APPROVED** [review]
Thanks for cleaning this up! LGTM once the changelogs are reverted.

File: packages/camera/camera_android/CHANGELOG.md (hunk 1 above)
Comment (FXschwartz) [reply packages/camera/camera_android/CHANGELOG.md]: Done!

Comment (tarrinneal) [reply packages/camera/camera_android/CHANGELOG.md]: > These changelog changes can all be reverted.

I think he means all changelogs can be reverted.

Comment (FXschwartz) [reply packages/camera/camera_android/CHANGELOG.md]: > > These changelog changes can all be reverted.
> 
> I think he means all changelogs can be reverted.

Ahhh, my mistake. All changelog changes have been reverted!

**Review by tarrinneal — APPROVED** [review]

**Pull request merged**